pkgs/top-level/all-packages.nix

```diff
@@ -2903,9 +2903,10 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security AppKit CoreServices;
     electron = electron_13;
   };
-  element-desktop-wayland = element-desktop.override {
-    useWayland = true;
-  };
+  element-desktop-wayland = writeScript "element-desktop-wayland" ''
+    #!/bin/sh
+    NIXOS_OZONE=1 exec ${element-desktop}/bin/element-desktop "$@"
+  '';
```

Comment:
I think we can move those to aliases.nix.

Reply:
@SuperSandro2000 I could add a deprecation notice on stderr? And then we remove these in 22.05?